Scenes/Worlds/StageOne.tscn

```diff
@@ -1,27 +1,19 @@
-[gd_scene load_steps=36 format=3 uid="uid://c1gk4v3ulvpue"]
```

Comment:
Not really sure what these changes are. Only thing I did was changing name of one of the nodes from 'Node' to 'TofSpawner' and attaching script to 'StageOne' node. May it be caused by different Godot versions?

Reply:
hmm.. we should probably try to figure this out before merging it

Reply:
Ok, fixed it, must have changed sth in the editor by accident. Now only my changes should be visible